index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Nelson Mandela</title>
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <!--
+
+    Hello Camper!
+
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
+    <main id="main">
+      <h1 id="title">Nelson Mandela</h1>
+      <p>The man who saved South Africa</p>
+
+      <div id="img-div">
+        <img id="image" src="images/mandela.jpeg" alt="mandela" />
+        <div id="img-caption">Describe picture</div>
+      </div>
+
+      <div id="tribute-info">
+        Lorem ipsum dolor, sit amet consectetur adipisicing elit. Alias ipsam
+        illo sed iusto corrupti eveniet perferendis facere fugit, a, officia
+        reprehenderit quos quam odio, perspiciatis aliquid impedit delectus.
+        Blanditiis eius dolores dolor necessitatibus sapiente laborum pariatur
+        odit sit deleniti maiores quidem voluptatum accusamus iste labore velit,
+        veritatis rerum a ipsam. Mollitia beatae ut suscipit pariatur rerum
+        laboriosam adipisci, natus quos distinctio, facilis autem expedita
+        veritatis eveniet officiis repellat aliquam iusto quisquam perspiciatis
+        sed nemo repellendus voluptate! Perspiciatis aliquam nam sint, nihil
+        dolor, quod cum voluptatum officia ut, distinctio ullam assumenda
+        possimus dignissimos delectus itaque a? Aspernatur quisquam sint id
+        culpa!
+      </div>
```

Comment:
You could have used `<p class="className">` here as stated in the [HTML & CSS best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)